nautobot/docs/release-notes/version-1.4.md

```diff
@@ -0,0 +1,253 @@
+
+#### Improved Filter Coverage for DCIM, Virtualization Models
+
+- DCIM: [#1729](https://github.com/nautobot/nautobot/issues/1729)
+- Virtualization: [#1735](https://github.com/nautobot/nautobot/issues/1735)
+
+The DCIM, Virtualization FilterSets have been updated with over 150 new filters, including hybrid filters that support filtering on both `pk` and `slug` (or `pk` and `name` where `slug` is not available). A new filter class `NaturalKeyOrPKMultipleChoiceFilter` was added to `nautobot.utilities.filters` to support filtering on multiple fields of a related object. See the [Best Practices](../development/best-practices/#mapping-model-fields-to-filters) documentation for more information.
+
+#### Job Hooks ([#2103](https://github.com/nautobot/nautobot/pull/2103))
+
+Jobs can now be configured to run automatically when a change event occurs on a Nautobot object. Job hooks associate jobs to content types and actions to run jobs when a create, update or delete action occurs on the selected content type. A new job base class `JobHookReceiver` was introduced that jobs must subclass to be associated with a job hook. See the [Job Hooks](../additional-features/job-hooks/) documentation for more information.
+
+#### Job Re-Runs ([#1875](https://github.com/nautobot/nautobot/issues/1875))
+
+JobResult records now save the arguments with which the Job was called, allowing for easy re-execution of the Job with the same arguments as before. A "re-run" button has been added to the JobResult list view and detail view.
+
+#### Location Data Model ([#1052](https://github.com/nautobot/nautobot/issues/1052))
+
+To locate network information more precisely than a Site defines, you can now define a hierarchy of Location Types (for example, `Building` ← `Floor` ← `Room`) and then create Locations corresponding to these types within each Site. Data objects such as devices, prefixes, VLAN groups, etc. can thus be mapped or assigned to Location representing a specific building, wing, floor, room, etc. as appropriate to your needs.
+
+!!! info
+    At present, Locations fill the conceptual space between the more abstract Region and Site models and the more concrete Rack Group model. In a future Nautobot release, some or all of these other models may be collapsed into Locations. That is to say, in the future you might not deal with Regions and Sites as distinct models, but instead your Location Type hierarchy might include these higher-level categories, becoming something like Country ← City ← Site ← Building ← Floor ← Room.
+
+#### Object Detail Tabs ([#1000](https://github.com/nautobot/nautobot/issues/1000))
+
+A plugin may now define extra tabs which will be appended to the object view's list of tabs.
+
+You can refer to the [plugin development guide](../plugins/development.md##adding-extra-tabs) on how to add tabs to existing object detail views.
+
```

Comment:
This feature should probably be highlighted in the release overview
```suggestion

#### Override Core and Plugin Views ([#1466](https://github.com/nautobot/nautobot/issues/1466))

Plugins can now override any of the core or plugin views by providing an `override_views` `dict` in a plugin's `views.py` file. For detailed information see the [plugin development guide](../plugins/development/#replacing-views)

```

Reply:
This isn't a hallmark feature that people should do very often. @lampwins Thoughts on including this in the release overview?